### PR TITLE
Fix bug 1278826: filter partially translated pluralized strings as missing

### PR DESCRIPTION
**Pull request by mathjazz**
This fixes the missing filter to also include partially translated pluralized strings.

@jotes r?

### CODE REVIEW

File: pontoon/base/models.py [hunk 1]
```diff
@@ -1087,7 +1087,7 @@ def with_status_counts(self, locale):
 
     def missing(self, locale):
         return self.with_status_counts(locale).filter(
-            Q(approved_count=0) & Q(fuzzy_count=0) & Q(suggested_count=0)
+            Q(approved_count__lt=F('expected_count')) & Q(fuzzy_count__lt=F('expected_count')) & Q(suggested_count__lt=F('expected_count'))
```
Comment (jotes) [inline pontoon/base/models.py]: A relative nit: I think we don't test case for this behaviour....

**Comment by jotes** [comment]
r+wc

File: pontoon/base/models.py (hunk 1 above)
Comment (mathjazz) [inline pontoon/base/models.py]: I'm afraid we don't test filters at all. :/

Comment (jotes) [inline pontoon/base/models.py]: Are you sure?
https://github.com/mozilla/pontoon/blob/master/pontoon/base/tests/test_models.py#L1079

Comment (mathjazz) [inline pontoon/base/models.py]: Thanks for introducing me to Pontoon! :-)

**Pull request merged**